WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/ShippingLabelFormViewModel.swift

```diff
@@ -24,3 +24,20 @@ final class ShippingLabelFormViewModel {
         return [Section(rows: rows)]
     }
 }
+
+// MARK: - Utils
+extension ShippingLabelFormViewModel {
+    func fromAddressToShippingLabelAddress(address: Address?) -> ShippingLabelAddress? {
```

Comment:
nit: instead of exposing `Order` and this function, could the view model provide the origin and destination address of type `ShippingLabelAddress` directly?

in either case, the new API can be unit tested? 🙂